validators/types_unix.go

```diff
@@ -66,10 +67,10 @@ var DefaultSysSpec = SysSpec{
 	CgroupsOptional: []string{
 		// The hugetlb cgroup is optional since some kernels are compiled without support for huge pages
 		// and therefore lacks corresponding hugetlb cgroup
-		"hugetlb",
+		"hugetlb", "blkio",
 	},
 	CgroupsV2:         []string{"cpu", "cpuset", "devices", "freezer", "memory", "pids"},
-	CgroupsV2Optional: []string{"hugetlb"},
+	CgroupsV2Optional: []string{"hugetlb", "blkio"},
```

Comment:
what about cgroups v1? are the container runtimes going to support this with v1 as well?
if not, this can be explained in the comment as well:
https://github.com/kubernetes/system-validators/pull/29/files#r812016832